contracts/test/v0.7/KeeperRegistry.test.ts

```diff
@@ -1608,8 +1661,8 @@ describe('KeeperRegistry', () => {
         )
       await linkToken.connect(owner).approve(registry.address, toWei('100'))
 
-      // max payment is .75 eth for this config - this spread will yield some eligible and some ineligible
-      const balances = ['0', '0.01', '0.1', '0.4', '0.7', '0.8', '1', '2', '10']
+      // max payment is .85 link for this config - this spread will yield some eligible and some ineligible
+      const balances = ['0', '0.01', '0.1', '0.7', '0.8', '0.9', '1', '2', '10']
```

Comment:
I looked into this because it seems to be a big change that didn't really require much to change. Can we make the assertions at the end a little tighter? Ideally we would specify an exact amount that we expect to pass. Maybe even break out the error cases from the non-error cases into another test.

Reply:
I cleaned up this test pretty considerably. We only needed two cases here: a funded one and an underfunded one.